File: Code/c/pattern6.c
#include<stdio.h>
void main()
{
    int limit, i, j;

    printf("Enter Limit: ");
    scanf("%d", & limit);

    for (i = 1; i < limit; i++)
    {
        for (j = 0; j < limit - i; j++)
            printf("  ");
        for (j = 0; j < i; j++)
            printf(" *");
        printf("\n");
    }
    for (i = limit; i > 0; i--)
    {
        for (j = 0; j < limit - i; j++)
            printf("  ");
        for (j = 0; j < i; j++)
            printf(" *");
        printf("\n");
    }
}

File: Code/java/pattern6.java
import java.util.Scanner;

class Pattern6
{
    public static void main(String[] args)
    {
        int limit, i;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Limit: ");
        limit = scanner.nextInt();

        for (i = 1; i < limit; i++)
            System.out.println("  ".repeat(limit - i) + " *".repeat(i));
        for (i = limit; i > 0; i--)
            System.out.println("  ".repeat(limit - i) + " *".repeat(i));

        scanner.close();
    }
}